R/estimate_risk.R

```diff
@@ -213,393 +220,495 @@ estimate_risk <- function(
     approach,
     variable = NULL,
     at = NULL,
-    ...) {
+    ...
+) {
   implausible <- 0.99999
   estimand <- match.arg(estimand)
-  link <- switch(
-    EXPR = estimand[1],
-    rr = "log",
-    rd = "identity")
-  if(link == "log")
+
+  if(estimand[1] == "rr") {
+    link = "log"
     possible_approaches <- as.character(as.list(
       args(risks::riskratio))$approach)[-1]
-  else
+  }
+  if(estimand[1] == "rd") {
+    link = "identity"
     possible_approaches <- as.character(as.list(
       args(risks::riskdiff))$approach)[-1]
+  }
+
   if(!(approach[1] %in% possible_approaches))
-    stop(paste0(
+    rlang::abort(paste0(
```

Comment:
Let's not introduce a dependency. I know, dplyr needs rlang, but ultimately trying to get away from that for this specific package. Same below